presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2723,6 +2729,48 @@ public void testGroupedExecution()
             assertQuery(colocatedOneGroupAtATime, chainedSharedBuildOuterJoin, expectedChainedOuterJoinResult, assertRemoteExchangesCount(2));
             assertQuery(colocatedOneGroupAtATimeDynamic, chainedSharedBuildOuterJoin, expectedChainedOuterJoinResult, assertRemoteExchangesCount(2));
 
+            //
+            // Window function
+            // ===============
+            // Window partition key covers bucket key
+            @Language("SQL") String expectedWindowFunctionResult = format("VALUES %s", String.join(", ", nCopies(15000, "(1)")));
+            assertQuery(
+                    colocatedOneGroupAtATime,
+                    "SELECT rank() OVER (PARTITION BY key) FROM test_window",
```

Comment:
These tests looks quite trivial since they always partition on a column with unique values. 

Can we see if there are any test queries for window function that we can use here?

It looks like for other use case (JOIN/AGGREGATION), we never don't test the situation where it's not suitable for grouped execution (supposedly, they are covered by other integration tests). So I guess we don't need the tests `Window partition key does not cover bucket key`.